File: FizzBuzz/FizzBuzz.cpp
#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main()
{
    for (int i = 1; i < 101; i++)
    {
        string line;
        if (i % 3 == 0)
            line += "Fizz";
        if (i % 5 == 0)
            line += "Buzz";
        if (i % 3 && i % 5)
            line = static_cast<ostringstream*>(&(ostringstream() << i))->str();
        cout << line << endl;
    }
    return 0;
}

File: FizzBuzz/FizzBuzz.java
public class FizzBuzz
{
    public static void main(String args[])
    {
        for (int i = 1; i < 101; i++)
        {
            String line = "";
            if (i % 3 == 0)
                line += "Fizz";
            if (i % 5 == 0)
                line += "Buzz";
            if (line.equals(""))
                line = Integer.toString(i);
            System.out.println(line);
        }
    }
}
